adal/src/telemetry/java/com/microsoft/aad/adal/HttpEvent.java

```diff
@@ -87,20 +90,108 @@ void setRequestIdHeader(final String requestIdHeader) {
         setProperty(EventStrings.REQUEST_ID_HEADER, requestIdHeader);
     }
 
+    void setSpeRingInfo(final String xMsCliTelem) {
+        // if the header isn't present, do nothing
```

Comment:
Not important, but since SPE ring is only one of the elements, I'd consider renaming this method to something more generic.

Reply:
Agree. Will change 👍